source/state_representation/test/tests/test_cartesian_state.cpp

```diff
@@ -415,6 +416,7 @@ TEST(CartesianStateTest, TestPoseDistance) {
   EXPECT_LT(abs(d3 - 3.14159), 1e10 - 3);
   double d4 = dist(p2, p3);
   EXPECT_LT(abs(d3 - d4), 1e10 - 3);
+  EXPECT_FALSE(std::isnan(p4.dist(p4, CartesianStateVariable::ORIENTATION)));
```

Comment:
would be nice to add a real-value test case (for both position and orientation), not just LT / GT

Reply:
Oh.. I guess they are there but it's a bit confusing. EXPECT_NEAR would be more appropriate